opencl/extensions/public/cl_gl_private_intel_structures.h

```diff
@@ -48,15 +48,15 @@ typedef struct _tagGLCLResourceInfo {
 } GL_CL_RESOURCE_INFO, *PGL_CL_RESOURCE_INFO;
 
 typedef struct _tagCLGLBufferInfo {
```

Comment:
this is struct shared with GL, we have it in a form that GL is expressing. This cannot be optimized

Reply:
> this is struct shared with GL, we have it in a form that GL is expressing. This cannot be optimized

fixed it https://github.com/intel/compute-runtime/pull/786/commits/67c2077a547547bcea076e379191dc18e09e12f5